Source/Controllers/Other/SPBundleManager.m

```diff
@@ -215,33 +213,38 @@ - (void)renameLegacyBundles{
 
             NSError *error = nil;
 
+            // Check if a migrated version already exists
             if ([fileManager fileExistsAtPath:migratedPath isDirectory:nil]) {
-                SPLog(@"File exists at path: %@", migratedPath);
+                SPLog(@"Migrated version already exists at path: %@", migratedPath);
+                // Remove the old legacy bundle since we already have a migrated version
+                NSError *removeError = nil;
+                if (![fileManager removeItemAtPath:legacyPath error:&removeError]) {
```

Comment:
This is the key - looks like we weren't actually removing the old bundles that had already been migrated